### PR TITLE
fastify-cors & account signature validation

### PR DESCRIPTION
**Pull request by marie-fourier**
- Added account & paymaster signatature validation
- Added fastify-cors
- Any GET request will be responded with the text `GET requests are not supported. Visit https://docs.etherspot.io/etherspot-skandha-bundler`

Tested against https://github.com/eth-infinitism/bundler-spec-tests/tree/94a0634ebd6e6da657e863c134d20bc46724ed0f
With EP v0.6.0

image

### CODE REVIEW

File: packages/api/package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "check-readme": "typescript-docs-verifier"
   },
   "dependencies": {
+    "@fastify/cors": "^8.2.1",
```
Comment (ch4r10t33r) [inline packages/api/package.json:34]: Can we use an exact version please?

File: packages/api/src/server.ts [hunk 2]
```diff
@@ -40,28 +42,23 @@ export class Server {
...
 
-  version(): void {
-    const { version } = JSON.parse(readFileSync(packageJson).toString());
-    this.app.get("/version", () => {
-      return {
-        version,
-      };
+  setup(): void {
+    this.app.get("*", () => {
+      return "GET requests are not supported. Visit https://docs.etherspot.io/etherspot-skandha-bundler";
```
Comment (ch4r10t33r) [inline packages/api/src/server.ts:57]: Didnt we look to support both? Or is POST the only recommended?

**Review by ch4r10t33r — CHANGES_REQUESTED** [review]

File: packages/api/src/server.ts (hunk 2 above)
Comment (marie-fourier) [reply packages/api/src/server.ts:57]: POST is the only option for JSON-RPC services afaik

File: packages/api/package.json (hunk 1 above)
Comment (marie-fourier) [reply packages/api/package.json]: yes, done

**Review by ch4r10t33r — APPROVED** [review]
LGTM

**Review by arddluma — APPROVED** [review]
LGTM 🚀

**Pull request merged**